sources/commands/option_external.c

```diff
@@ -49,13 +73,14 @@ int	option_external(t_control control, t_treenode *node, t_param *param)
 	*control.found = 1;
 	if (param->debug_output_level & DBG_EXTERNAL_SEARCH_FOLDERS)
 	{
-		printf("searching for command in folders:\ncurrent\n");
+		printf("searching for command in folders:\n");
```

Comment:
Current working folder is element 0 in PATH for the purpose of this minishell. We don't store it separately. This was it's easier to navigate while looking for the executable.

Reply:
@ioannnikulin could you please make more detailed description to PRs? then i would less time spend to merge it